CHANGELOG.md

```diff
@@ -4,6 +4,14 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
+## Unreleased Phase 2
+
+### Added
+
+- **CUMULUS-2703**
```

Comment:
Even though it is covered by the API documentation update, we should probably added a `### Notable changes` header and call out that `ORCA Backup` is now a supported `reportType` for the `POST /reconciliationReports` endpoint